charts/backstage/values.yaml

```diff
@@ -116,23 +116,111 @@ backstage:
   command: ["node", "packages/backend"]
   args: []
   extraAppConfig: []
-  extraContainers: []
+  # extraAppConfig:
```

Comment:
not sure if this was meant to make it into the PR?

Reply:
Hi @ChrisJBurns I left this in as an example of how to use in the values.yaml for different environments with a complete configuration. I can reduce this down to a few lines ?